.github/workflows/tests.yml

```diff
@@ -506,6 +506,19 @@ jobs:
           - arrangement: workers
             database: Postgres
 
+          - arrangement: monolith
+            database: SQLite
+            reactor: asyncio
+
+          - arrangement: monolith
+            database: Postgres
+            reactor: asyncio
+
+          - arrangement: workers
+            database: Postgres
+            reactor: asyncio
```

Comment:
this would normally be fine, but reality is going to kick in as this will double the number of runners needed to run Complement CI. We already run out of runners sometimes so that is possibly too much. :/

Reply:
This was a holdover from the previous work. I recommend removing it in the first comment. It's only here to show the reactor works
>I think the extra CI runners should be removed before merging, as that seems to be a scarce resource(as referenced from the previous PR). A follow-up PR can add a nightly or weekly scheduled run with this active.

Reply:
It's gone now.